evap/evaluation/models.py

```diff
@@ -382,7 +382,7 @@ class Evaluation(LoggedModel):
     # Disable to prevent editors from changing evaluation data
     allow_editors_to_edit = models.BooleanField(verbose_name=_("allow editors to edit"), default=True)
 
-    evaluation_evaluated = Signal(providing_args=['request', 'semester'])
+    evaluation_evaluated = Signal()
```

Comment:
Would it make sense to put these in a comment so people potentially receiving the signal can quickly figure out what arguments to expect?